site/layouts/shortcodes/note.html

```diff
@@ -1,6 +1,6 @@
 <aside class="note">
-    <div class="note-icon">
-        {{partial "svg/exclamation.svg" (dict "size" "20px" ) }}
+    <div class="note-content">
+        <strong>{{ .Get 0 }}</strong><br>
```

Comment:
We used the shortcode in the past like this parameter had any affect. This just fixes the behaviours.